Makefile

```diff
@@ -4,4 +4,9 @@ int:
 unit:
 	cargo test --locked  --no-fail-fast --lib
 
-test: unit int
\ No newline at end of file
+ex:
+	cargo test --examples
+	cargo test --example json_to_edn --features "json"
+	cargo run --example async --features "async"
+
+test: unit int ex
```

Comment:
What do you think of we using `integration` and `examples` instead of `int` and `ex` on the `Makefile`?

We can use `.PHONY: examples` for example, to don't clash with file names :slightly_smiling_face:

Reply:
Third time is the charm 33a4851